### PR TITLE
mgr: dashboard code cleanup

### PR DESCRIPTION
**Pull request by jcsp**

### CODE REVIEW

File: src/pybind/mgr/dashboard/servers.html [hunk 1]
```diff
@@ -24,9 +24,16 @@
             };
 
             rivets.formatters.short_version = function(version) {
-                // Turn "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
-                // into "1.2.3-g9asdasd"
-                return /ceph version (.+) \(.+\)$/.exec(version)[1];
+                // Expect "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
+                result = /ceph version (.+) \(.+\)/.exec(version);
```
Comment (dillaman) [inline src/pybind/mgr/dashboard/servers.html]: Nit: you should use ```/ceph version ([^ ]+) \(.+\)/```

File: src/pybind/mgr/dashboard/module.py [hunk 2]
```diff
@@ -577,6 +577,9 @@ def servers(self):
                 )
 
             def _servers(self):
+                log.info("Listing servers...")
```
Comment (dillaman) [inline src/pybind/mgr/dashboard/module.py]: Nit: debug?

File: src/pybind/mgr/dashboard/servers.html (hunk 1 above)
Comment (liewegas) [inline src/pybind/mgr/dashboard/servers.html]: can we make a structured ceph version thing?  i was considering changing the format of this string a couple weeks ago and didn't out of fear of things like this.  but we have 2 new fields added at the end now (release name, e.g. 'luminous') and release type ('dev', 'rc', 'stable').

Comment (liewegas) [inline src/pybind/mgr/dashboard/servers.html]: (that can come later, doesn't need to hold up this pr)

Comment (jcsp) [inline src/pybind/mgr/dashboard/servers.html]: The places we pass it around (e.g. daemon metadata) we only have a string.

We could improve the format of the string though -- prefixing with "ceph version" seems redundant.  Once that's removed we could guide people to take everything up to the first space as the short version and treat everything else as detail, perhaps.

@dillaman regex updated

File: src/pybind/mgr/dashboard/module.py (hunk 2 above)
Comment (jcsp) [inline src/pybind/mgr/dashboard/module.py]: fixed.

**Pull request merged**